meta/documents/user_guide/en/initial_setup.md

```diff
@@ -81,37 +81,37 @@ This procedure is required only if [the `Send all inventory items to Wayfair` se
 
 1. From the main Plentymarkets page, go to `Item` >> `Edit item`
 
-2. Search for item(s) and open them
+2. Search for item(s) and open them Ensure that item says **`main variation`** see image below

```

Comment:
The fragment at the end of this is probably meant to be a new step or sub-step?
Also, there's no punctuation or separation between ideas.

Reply:
it is more of a note to suppliers, I'll a the note keyword